odk1-src/briefcase-using.rst

```diff
@@ -37,43 +37,41 @@ Briefcase will ask you to provide the following information when setting a Centr
 Aggregate server
 ~~~~~~~~~~~~~~~~
 
-Briefcase will ask you to provide the following information when setting an Aggregate server as the pull source:
+Briefcase will ask for the following information when setting an Aggregate server as the pull source:
 
 - A :guilabel:`URL`
-- An optional :guilabel:`Username`
-- An optional :guilabel:`Password`
+- A :guilabel:`Username` (optional)
+- A :guilabel:`Password` (optional)
 
 .. _pull-from-collect:
 
-Collect storage folder
-~~~~~~~~~~~~~~~~~~~~~~
+Collect directory
+~~~~~~~~~~~~~~~~~
 
-Briefcase will ask you to choose a directory in your computer where you have downloaded Collect's storage folder. We recommend to follow this steps to get a copy of Collect's storage folder into your computer:
+Briefcase will ask for the directory on your computer where you have placed Collect's :file:`/odk` directory. We recommend following these steps to get a copy of Collect's :file:`/odk` directory into your computer:
 
-#. Ensure all filled-in forms are finalized.
+1. Ensure all filled-in forms are finalized.
 
   If you have incomplete forms that you cannot finalize before pulling into Briefcase, delete them. If you need to keep them, make a copy of :file:`/sdcard/odk` before deleting them, and restore it after you are finished.
 
-#. Using your device, create a zip archive of the entire :file:`odk` directory with an app such as `OI File Manager <https://play.google.com/store/apps/details?id=org.openintents.filemanager>`_.
-
-#. Connect your Android device to your computer using a USB cable and choose to mount it as a Media device.
-#. Copy the zip file you created from the Android device to your local hard drive.
-#. Once it is copied onto your local hard drive, unzip the file.
+2. Using your device, create a zip archive of the entire :file:`odk` directory with a file managing app such as `OI File Manager <https://play.google.com/store/apps/details?id=org.openintents.filemanager>`_.
+3. Transfer the zip file to your local hard drive via a USB cable. You can also use the Share feature in your file manager to transfer it to a third-party service like Google Drive then download it to your local hard drive.
+4. Once the zip file is on your local hard drive, unzip the file.
 
 .. warning::
 
-  Briefcase cannot discriminate between duplicate submissions. After you pull completed forms into Briefcase, it is important that you delete them from Collect. Otherwise, the next time you pull in forms, you will create duplicates.
+  Briefcase cannot discriminate between duplicate forms. After you pull completed forms into Briefcase, it is important that you delete them from Collect. Otherwise, the next time you pull in forms, you will create duplicates.
```

Comment:
I think this should really be submissions.

Reply:
I used forms because it's not really a submission. Briefcase pulls every instance.